magento/magento2
# Fixing Colorpicker Definition for RequireJS Use

Before this we were unable to require the colorpicker as a dependency in a custom js component using requirejs.

### Description

Simply added a string ID for the colorpicker in the define method in `lib/web/jquery/colorpicker/js/colorpicker.js`

### Fixed Issues (if relevant)

1. magento/magento2#
#17008: Core Colorpicker JS doesn't work with RequireJS

### Manual testing scenarios

1. Create custom js file that requires the colorpicker
````
// Vendor_Module/js/this/custom/file
require([
    'jquery',
    'jquery/colorpicker/js/colorpicker',
], function ($) {
    'use strict';
    // do anything, the damage is done
    console.log("required the colorpicker. Be free!");
});
````
2. Make sure the message "required the colorpicker. Be free!" is logged.

### Contribution checklist
